client/components/domains/domain-suggestion/test/index.js

```diff
@@ -27,16 +30,44 @@ describe( 'Domain Suggestion', function() {
 	} );
 
 	describe( 'has attributes', () => {
-		it( 'should have data-e2e-domain attribute for e2e testing', () => {
-			const domainSuggestion = shallow( <DomainSuggestion
-				domain="example.com" isAdded={ false }/> );
-			if ( domainSuggestion.props()[ 'data-e2e-domain' ] ) {
-				expect( domainSuggestion.props()[ 'data-e2e-domain' ] ).to.equal( 'example.com' );
-			} else {
+		context( 'AB test clickableButton', () => {
```

Comment:
Personally I prefer using just nested `describe()`. But I've seen `context()` being used in many locations as well.